src/Stoa.ts

```diff
@@ -423,6 +436,22 @@ class Stoa {
             }
         });
     }
+
+    /**
+     * Get the maximum number of blocks that can be recovered at one time
+     */
+    public get max_count_recovery (): number
+    {
+        return this._max_count_recovery;
+    }
+
+    /**
+     * Set the maximum number of blocks that can be recovered at one time
+     */
+    public set max_count_recovery (value: number)
+    {
+        this._max_count_recovery = value;
+    }
```

Comment:
```suggestion
    get max_count_recovery (): number
    {
        return this._max_count_recovery;
    }

    /**
     * Set the maximum number of blocks that can be recovered at one time
     */
    set max_count_recovery (value: number)
    {
        this._max_count_recovery = value;
    }
```
I don't think `Getter/Setter` needs a `public`.